railties/test/application/rake/single_resource_test.rb

```diff
@@ -0,0 +1,30 @@
+require "isolation/abstract_unit"
+require "active_support/core_ext/string/strip"
+class RakeTest < ActiveSupport::TestCase
+  include ActiveSupport::Testing::Isolation
+  def setup
+    build_app
+  end
+
+  def teardown
+    teardown_app
+  end
+  def test_26606
+    app_file "config/routes.rb", <<-RUBY
+      Rails.application.routes.draw do
+        resource :post
+      end
+    RUBY
+    expected_output = ["   Prefix Verb   URI Pattern          Controller#Action",
+                       " new_post GET    /post/new(.:format)  posts#new",
+                       "edit_post GET    /post/edit(.:format) posts#edit",
+                       "     post GET    /post(.:format)      posts#show",
+                       "          PATCH  /post(.:format)      posts#update",
+                       "          PUT    /post(.:format)      posts#update",
+                       "          DELETE /post(.:format)      posts#destroy",
+                       "          POST   /post(.:format)      posts#create\n"].join("\n")
+
+    output = Dir.chdir(app_path) { `bin/rails routes -c PostController` }
+    assert_equal expected_output, output
+  end
+end
```

Comment:
When I asked to add a test case the test at ` railties/test/application/rake_test.rb` were not changed yet. Now that it was I think we can remove this new file, but I can do it since you already had many rounds of reviews 😄